src/annotator/highlighter.js

```diff
@@ -246,11 +246,11 @@ export function highlightRange(range, cssClass = 'hypothesis-highlight') {
   const whitespace = /^\s*$/;
   textNodeSpans = textNodeSpans.filter(span =>
     // Check for at least one text node with non-space content.
-    span.some(node => !whitespace.test(node.nodeValue))
+    span.some(node => !whitespace.test(node.data))
```

Comment:
For `Text` nodes, `nodeValue` and `data` are the same. The difference is that `nodeValue` is nullable because it is part of the `Node` interface, whereas `data` is specific to `CharacterData` nodes (text, comments, cdata) that definitely have text.